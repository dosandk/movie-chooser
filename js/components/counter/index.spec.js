import React from 'react';
import Counter from './index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Counter/>).toJSON();

  expect(tree).toMatchSnapshot();
});
