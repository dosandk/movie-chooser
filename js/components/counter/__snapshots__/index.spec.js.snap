// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    Clicked: 
     times
     
    <button
      onClick={undefined}
    >
      +
    </button>
     
    <button
      onClick={undefined}
    >
      -
    </button>
     
    <button
      onClick={undefined}
    >
      Increment if odd
    </button>
  </div>
  <div>
    <h3>
      Loader
    </h3>
    <button
      onClick={undefined}
    >
      show temporary
    </button>
    <button
      onClick={undefined}
    >
      show loader
    </button>
    <button
      onClick={undefined}
    >
      hide loader
    </button>
  </div>
</div>
`;
